roles/ffh.mesh_wireguard/tasks/helper.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Create directory for wireguard helpers
+  file:
+    path: "{{ wg_helper_dir }}"
+    state: directory
+
+- name: Install vx.py
+  template:
+    dest: "{{ wg_helper_dir }}/vx.py"
```

Comment:
Do we still need this script on the server?

Reply:
None of the two wg roles needs this for now.

Reply:
And as it's in ansible/scripts I think It can go.

Reply:
gone since 2ef41a02bfa0c83d08a5804d22b8d6439560ac1f